src/test/clojure/jarl/builtins/conversions_test.clj

```diff
@@ -0,0 +1,13 @@
+(ns jarl.builtins.conversions-test
+  (:require [clojure.test :refer [deftest]]
+            [test.utils :refer [testing-builtin]]))
+
+(deftest to-number-test
```

Comment:
No tests for deprecated built-ins?
Then we should definitely log a warning when such functions are used.